src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -568,6 +568,12 @@ struct SvgNodeIdPair
     char *id;
 };
 
+struct EmbeddedFont
+{
+    char* name;
+    char* src;
```

Comment:
https://www.w3.org/TR/css-fonts-3/#at-font-face-rule
We only support ttf type, but font-face can have multiple src. It is not necessary now, but it seems that we should consider cases where multiple src are applied.

Reply:
I don't think it's about what types we support. It's true that src can contain multiple urls (including local sources, etc.):
```
@font-face {
            font-family: 'MyFont';
            src: url('data:font/ttf;base64,...data....') format('truetype'),       
                   url('data:font/ttf;base64,...data....') format('truetype');
            }
```
However, we currently don't validate whether a given font supports the specified text.
We also don't support `font-family` as a prioritized list of font family names - only a single name is supported.

I'd prefer to handle this after a real-case scenario arises. Is it ok with you?

Reply:
> I'd prefer to handle this after a real-case scenario arises. Is it ok with you?

Sure, let's handle when a real case to arise.